uniffi_bindgen/src/bindings/python/templates/NamespaceLibraryTemplate.py

```diff
@@ -5,19 +5,19 @@
 # E.g. we might start by looking for the named component in `libuniffi.so` and if
 # that fails, fall back to loading it separately from `lib${componentName}.so`.
 
-def loadIndirect(componentName):
+def loadIndirect():
```

Comment:
It's a shame this is done here - eg, the "win" branch isn't really correct in most scenarios - a `lib_` prefix is the exception rather than the norm and tends to reflect someone who came from a non-windows background - eg, my "system32" directory has 3500 dll files, which exactly zero starting with "lib_" :)

(I've no problem with going ahead with this though, just thought I'd mention it)

Reply:
@mhammond FYI I didn't see your comment before my approval 😬

Reply:
Thanks for the heads-up; I'm sure you can guess how many times I've tried running this on Windows in practice given our current need on that platform, which is a shame :-/

Reply:
I'm going to push ahead with merging this, but I'm also installing the native Windows Rust build toolchain right now and will poke at it in between some other things, ref https://github.com/mozilla/uniffi-rs/issues/391

Reply:
> @mhammond FYI I didn't see your comment before my approval 😬

No probs - I didn't intend for my comment to block approval!